src/site/_includes/partials/post-next.njk

```diff
@@ -177,7 +177,7 @@
           </a>
         </div>
       {% endif %}
-      <div class="docked-actions flow flow-space-base">    
+      <div class="docked-actions flow flow-space-base">
```

Comment:
The action buttons are misaligned on small screens:
<img width="425" alt="Screenshot 2022-02-01 at 13 13 38" src="https://user-images.githubusercontent.com/1914261/151966621-7c9de70a-f521-4a54-a9a7-e1d48a6ff405.png">